CMakeLists.txt

```diff
@@ -11,20 +11,20 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 # on clang and g++
 if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
   if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "11.0.0")
-      MESSAGE(FATAL_ERROR, "G++ versions older than 11.0 are not supported by QLever")
+      MESSAGE(FATAL_ERROR "G++ versions older than 11.0 are not supported by QLever")
   else()
       add_compile_options(-fcoroutines)
   endif()
 
 elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
     if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "14.0.0")
-        MESSAGE(FATAL_ERROR, "Clang++ versions older than 14.0 are not supported by QLever")
+        MESSAGE(FATAL_ERROR "Clang++ versions older than 14.0 are not supported by QLever")
     elseif(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "16.0.0")
         add_compile_options(-fcoroutines-ts)
         # starting from Clang 16, coroutines are enabled by default for C++ 20
     endif()
```

Comment:
Thanks for finding this subtlety. While you are at it, you can change this section to reflect the truth:
We in the meantime require clang versions of at least 16.0,so the `14` can be changed to `16` and the `elseif` branch can be completely eliminated.